### PR TITLE
Add Brier score loss to classification

### PR DESCRIPTION
**Pull request by nicholasdomene**
#### Describe the changes you've made
I would like to propose the addition of Brier score as classification metric.
https://scikit-learn.org/stable/modules/generated/sklearn.metrics.brier_score_loss.html

## Type of change
Adding new metric to classification models

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] My changes generate no new warnings.

### CODE REVIEW

**Review by ngupta23 — DISMISSED** [review]

**Review by ngupta23 — COMMENTED** [review]
Could you check again. The unit tests are failing.

**Comment by nicholasdomene** [comment]
Multiclass test is running Brier eventhough metric is declared as is_multiclass = False. I will look deeper into this

**Comment by moezali1** [comment]
@nicholasdomene I don't think so we should add this as the default metric. There are already 6 metrics.

Even without adding it as default, users can add the metric using the `add_metric` function if they would like.

```
from sklearn.metrics import brier_score_loss

from pycaret.classification import *
s = setup(data, target)
add_metric('brier', 'brier', brier_score_loss)
...
...
```
Please feel free to open a thread on our discussion forum. If there is enough upvote to include brier as a default metric I will happy to consider this in the future releases.
